jsf/multi-jsf-installer/src/main/resources/mojarra-api-module.xml

```diff
@@ -36,7 +36,7 @@
     </dependencies>
 
     <resources>
-        <resource-root path="jsf-api-${jsf-version}.jar"/>
+        <resource-root path="jboss-jsf-api_2.3_spec-${jsf-version}.jar"/>
```

Comment:
This change will cause issues when attempting to install Mojarra 2.2.x or Mojarra 2.1.x. We may need a separate mojarra-api-module.xml template for Mojarra 2.3.x.